stories/documentation/forms/textfields/textfield-clearable.stories.ts

```diff
@@ -12,7 +12,9 @@ function getTemplate(args: TextfieldClearableStory): string {
 	<label class="textfield mod-clearable">
 		<input class="textfield-input" type="text" placeholder="Placeholder">
 		<span class="textfield-label">Label</span>
-		<a href="#" role="button" class="actionIcon textfield-actionClear"><span aria-hidden="true" class="lucca-icon icon-cross"></span></a>
+		<button class="clear textfield-clear">
+		  <span aria-hidden="true" class="lucca-icon icon-close"></span>
+		</button>
```

Comment:
I don't think you can put a `button` inside a `label` that also contains an `input`.

Reply:
Indeed. Fixed.